ui/apps/platform/src/Containers/Policies/Wizard/Step2/PolicyBehaviorForm.tsx

```diff
@@ -166,25 +171,47 @@ function PolicyBehaviorForm({ hasActiveViolations }: PolicyBehaviorFormProps) {
-                            evaluated in the build pipeline.
+                            <strong>Build stage</strong> policies can inspect only images built in
+                            the build pipeline, using criteria related to the image registry,
+                            content, vulnerability data and scanning process. When enforced, policy
+                            violations may be used to fail the build.
                         </p>
                         <p>
-                            <strong>Deploy stage</strong> policies can inspect workloads and/or
-                            their images. They are evaluated while creating or updating a workload
-                            resource, and re-evaluated periodically or on demand.
+                            <strong>Deploy stage</strong> policies can inspect workload
+                            configurations and/or their images. They are evaluated while creating or
+                            updating a workload resource and re-evaluated periodically or on demand.
+                            When enforced, policy violations result in rejection of workload
+                            admission or update, or if admitted, workload replicas being scaled down
+                            to zero.
                         </p>
                         <p>
-                            <strong>Build and Deploy stage</strong> policies are a convenient option
-                            to inspect images in both the build pipeline and during workload
-                            admission, and apply enforcement to either or both stages (in a single
-                            policy).
+                            <strong>Build+Deploy stage</strong> policies are a convenient option to
+                            inspect images in both the build pipeline and during workload admission
+                            and apply enforcement to either or both stages in a single policy.
                         </p>
                         <p>
-                            <strong>Runtime</strong> policies operate on one of two domains:
-                            Workload Activity or Kubernetes Resource Operations. The two domains are
-                            associated with different “Event Sources”.
+                            <strong>Runtime</strong> policies inspect either workload activity or
+                            Kubernetes resource operations, depending on the event source selected.
+                            When enforced, runtime policies that inspect workload activity terminate
+                            the offending pod. Enforcement is not available for the policies
+                            inspecting sensitive operations via the Kubernetes audit log.
                         </p>
+                        <div className="pf-v5-u-pt-md">
+                            Learn more about policy{' '}
```

Comment:
Bravo for link and extra credit for `ExternalLink` element which was omitted recently somewhere else.

Not change request, but for your information (pardon pun) in the future so we invent less and apply more.

IBM, Red Hat, Microsoft, and beyond, have generic pattern **For more information, see** with specific link text for us of **RHACS documentation** if the destination is product docs.

An optional phrase, for example, **about lifecycle stages** is usually redundant in the context.